src/app/index.js

```diff
@@ -34,8 +38,25 @@ export default class CGUs extends events.EventEmitter {
 
   async init() {
     if (!this._serviceDeclarations) {
+      this.trackDocumentChangesQueue = async.queue(async ({ serviceDocument }) => this.trackDocumentChanges(serviceDocument), NUMBER_OF_TRACK_DOCUMENT_WORKERS);
+      this.refilterDocumentsQueue = async.queue(async ({ serviceDocument }) => this.refilterAndRecordDocument(serviceDocument), NUMBER_OF_REFILTER_WORKERS);
+
+      const queueErrorHandler = (error, { serviceDocument }) => {
+        const { serviceId, document: { type } } = serviceDocument;
+        if (error instanceof InaccessibleContentError) {
+          return this.emit('inaccessibleContent', error, serviceId, type);
+        }
+        this.emit('error', error, serviceId, type);
+        this.trackDocumentChangesQueue.kill();
+        this.refilterDocumentsQueue.kill();
+      };
```

Comment:
Declare a method rather than an anonymous function in `init`?

Reply:
Why? I do not want to expose this function.